framework/src/modules/network/defaults/config.js

```diff
@@ -8,16 +8,13 @@ const defaultConfig = {
 			env: 'LISK_WS_PORT',
 			arg: '--port,-p',
 		},
-		address: {
+		hostAddress: {
```

Comment:
Why we need to change the config, not the usage? Similar name used in the http_api module as well, should we change it there as well?